migration/v2.21/prepare/12-ntp-servers.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+ROOT="$(readlink -f "${HERE}/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+yq_add sc k8s_cluster/ck8s-k8s-cluster .ntp_enabled true
+yq_add sc k8s_cluster/ck8s-k8s-cluster .ntp_manage_config true
+yq_add sc k8s_cluster/ck8s-k8s-cluster .ntp_servers '[ "gbg1.ntp.netnod.se iburst", "gbg2.ntp.netnod.se iburst", "lul1.ntp.netnod.se iburst", "lul2.ntp.netnod.se iburst", "mmo1.ntp.netnod.se iburst", "mmo2.ntp.netnod.se iburst", "sth1.ntp.netnod.se iburst", "sth2.ntp.netnod.se iburst", "sth3.ntp.netnod.se iburst", "sth4.ntp.netnod.se iburst", "svl1.ntp.netnod.se iburst", "svl2.ntp.netnod.se iburst"]'
+yq_add sc k8s_cluster/ck8s-k8s-cluster .ntp_timezone "\"Etc/UTC\""
+
+yq_add wc k8s_cluster/ck8s-k8s-cluster .ntp_enabled true
+yq_add wc k8s_cluster/ck8s-k8s-cluster .ntp_manage_config true
+yq_add wc k8s_cluster/ck8s-k8s-cluster .ntp_servers '[ "gbg1.ntp.netnod.se iburst", "gbg2.ntp.netnod.se iburst", "lul1.ntp.netnod.se iburst", "lul2.ntp.netnod.se iburst", "mmo1.ntp.netnod.se iburst", "mmo2.ntp.netnod.se iburst", "sth1.ntp.netnod.se iburst", "sth2.ntp.netnod.se iburst", "sth3.ntp.netnod.se iburst", "sth4.ntp.netnod.se iburst", "svl1.ntp.netnod.se iburst", "svl2.ntp.netnod.se iburst"]'
```

Comment:
There should be something here (maybe a prompt) to ask if the environment you are upgrading is hosted in Sweden. Because if not the operator needs to manually add other appropriate ntp servers.

Reply:
Added a prompt